sirepo/auth/github.py

```diff
@@ -7,16 +7,19 @@
 :license: http://www.apache.org/licenses/LICENSE-2.0.html
 """
 from __future__ import absolute_import, division, print_function
-from pykern.pkcollections import PKDict
 from pykern import pkconfig
 from pykern import pkinspect
+from pykern.pkcollections import PKDict
 from pykern.pkdebug import pkdc, pkdexc, pkdlog, pkdp
 from sirepo import api_perm
 from sirepo import auth
 from sirepo import auth_db
 from sirepo import cookie
+from sirepo import events
```

Comment:
Should I fix all of the other imports? I went with unqualified to keep the same pattern as already in use.

Reply:
not now. i think it is important for "events" to use sirepo.events, because it is so generic. cookie less so. auth_db not at all.

It's so funny that Perl has [TIMTOWTDI](https://en.wikipedia.org/wiki/There%27s_more_than_one_way_to_do_it), and Python is the same way.

Reply:
1a6d6e2b